product-service/src/functions/addProduct/handler.ts

```diff
@@ -0,0 +1,24 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { Client } from 'pg';
+import { dbOptions } from 'src/utils/db-options';
+
+const addProduct: ValidatedEventAPIGatewayProxyEvent<typeof Object> = async (event) => {
+  console.log('addProduct invoked --- ', event);
+  console.log('params --- ', event.body);
+
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  try {
```

Comment:
? why it is empty

Reply:
Oh. I forget about it. I finish it today

Reply:
Still empty :(